GTBitsOfGood/brain-exercise-analytics
# Vt/45 search and filter

## Implement Search Component

Issue Number(s): #45

What does this PR change and why?
Adds Search and Advanced Search functionality, to make searching/filtering patients easier

### Checklist

- [x]  Create the Search component
- [x]  Add Advanced Filters functionality
- [x]  Add functionality to Apply and Clear buttons
- [x]  Clicking Apply button should display new pink tags

### Critical Changes

- Search component
- Advanced Search component
- Tag component

### Related PRs

N/A

### Testing

1. Navigate to `/patient/search`
2. Click the `Advanced Filter` text
3. Input any information and click `Apply` or `Clear`
4. Close the `Advanced Filter` to see updated Tags
5. Click x on Tag to close it
